pkg/watcher/reconciler/dynamic/dynamic_test.go

```diff
@@ -353,16 +356,47 @@ func TestReconcile_TaskRun(t *testing.T) {
 		}
 	})
 
+	t.Run("do not delete object with owner references has PipelineRun object and owner check is disabled", func(t *testing.T) {
+		r.cfg.CheckOwner = false
+
+		taskrun.OwnerReferences = []metav1.OwnerReference{{
+			APIVersion: "tekton.dev/v1",
+			Kind:       "PipelineRun",
+			Name:       "test-pipelinerun",
+			UID:        types.UID(uuid.NewString()),
+		}}
+
+		// delete taskrun before creating
+		trclient.Delete(ctx, taskrun.GetName(), metav1.DeleteOptions{})
+
+		// Recreate the object to retest the deletion
+		if _, err := trclient.Create(ctx, taskrun, metav1.CreateOptions{}); err != nil {
+			t.Fatal(err)
+		}
+
+		if err := r.Reconcile(ctx, taskrun); err != nil {
+			t.Fatal(err)
+		}
+
+		// Make sure that the resource no longer exists
```

Comment:
Fix comment - resource should exist.
There are other identical comments that should be corrected - appears to be a copy/paste issue.

```suggestion
		// Make sure the TaskRun is not deleted
```

Reply:
Done